ui/app/AppLayouts/Chat/controls/community/PermissionItem.qml

```diff
@@ -0,0 +1,240 @@
+import QtQuick 2.0
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Utils 0.1
+
+//import utils 1.0
+
+Rectangle {
+    id: root
+
+    property var holdingsListModel: ListModel {}
+    property var permissionsObjectModel: QtObject {}
+    property var channelsListModel: ListModel {}
+    property bool isPrivate: false
+
+    signal editClicked
+    signal duplicateClicked
+    signal removeClicked
+
+    QtObject {
+        id: d
+        readonly property int flowRowHeight: 32
+    }
+
+    border.color: Theme.palette.baseColor2
+    border.width: 1
+    implicitHeight: header.height + content.implicitHeight + content.anchors.topMargin + content.anchors.bottomMargin + footer.implicitHeight + footer.anchors.bottomMargin
+    implicitWidth: 560 // by design
+    radius: 16
+
+    Rectangle {
+        id: header
+        color: Theme.palette.baseColor2
+        anchors.top: parent.top
+        width: parent.width
+        height: 34
+        radius: 16
+
+        RowLayout {
+            anchors.fill: parent
+            spacing: 8
+
+            StatusIcon {
+                Layout.leftMargin: 19
+                icon: "checkmark"
+                width: 11
+                height: 8
+            }
+
+            StatusBaseText {
+                Layout.fillWidth: true
+                text: qsTr("Active")
+                font.pixelSize: 15
+            }
+
+            StatusIcon {
+                Layout.rightMargin: 10
+                visible: root.isPrivate
+                icon: "hide"
+                color: Theme.palette.baseColor1
+            }
+        }
+    }
+
+    Flow {
+        id: content
+        anchors {
+            top: header.bottom
```

Comment:
For me here it makes sense to use anchors since we have clarely a `header`, `content` and `footer` but I can do it in another way, as well. Do you prefer to use a `ColumnLayout` instead?

Reply:
Definitely :)